nvflare/edge/device/android/sdk/training/TrainingArtifactsHelper.kt

```diff
@@ -0,0 +1,289 @@
+package com.nvidia.nvflare.sdk.training
+
+import android.content.Context
+import android.util.Log
+import java.io.File
+import java.text.SimpleDateFormat
+import java.util.*
+
+/**
+ * Utility class to help access and manage training artifacts.
+ * Provides easy access to saved models, logs, and training data.
+ */
+class TrainingArtifactsHelper(private val context: Context) {
```

Comment:
we either choose TrainingArtifactsManager or TrainingArtifactsHelper?
we should just use one class?